src/systems/ec/implementations/react/templates/compositions/navigation-lists/examples/Default.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Link from '@ecl/ec-react-component-link';
+import { UnorderedList, UnorderedListItem } from '@ecl/ec-react-component-list';
+
+export default () => (
+  <div>
+    <h2 className="ecl-u-ma-none ecl-u-type-prolonged-xl ecl-u-type-bold">
+      <Link
+        href="#example"
```

Comment:
This kind of target triggers accessibility error in storybook

Reply:
True, I will fix it 👍